internal/cli/atlas/performanceadvisor/namespaces/list.go

```diff
@@ -105,7 +104,9 @@ If you don't set the duration option or the since option, this command returns d
 	}
 
 	cmd.Flags().StringVar(&opts.HostID, flag.HostID, "", usage.HostID)
+	_ = cmd.Flags().MarkDeprecated(flag.HostID, "Flag is invalid for MongoDB Atlas")
```

Comment:
❤️ Thanks for helping our users here in case they were still using the flags in scripts

Reply:
yeah initially I was goign to delete then though it better to deprecate first